dtfs-central-api/src/v1/routes/utilisation-reports-routes.js

```diff
@@ -832,4 +828,53 @@ utilisationReportsRouter
   .all(validation.sqlIdValidation('reportId'), validation.sqlIdValidation('feeRecordId'), handleExpressValidatorResult)
   .put(validatePutFeeRecordCorrectionTransientFormDataPayload, putFeeRecordCorrectionTransientFormData);
 
+/**
+ * @openapi
+ * /utilisation-reports/:reportId/fee-records/:feeRecordId/correction-transient-form-data/:userId:
+ *   get:
+ *     summary: Gets the fee record correction transient form data against a user and fee record
+ *     tags: [Utilisation Report]
+ *     description: Gets the fee record correction transient form data against a user and fee record
+ *     parameters:
+ *       - in: path
+ *         name: reportId
+ *         schema:
+ *           type: string
+ *         required: true
+ *         description: the id for the report
+ *       - in: path
+ *         name: feeRecordId
+ *         schema:
+ *           type: string
+ *         required: true
+ *         description: the id for the fee record
+ *       - in: path
+ *         name: userId
+ *         schema:
+ *           type: string
+ *         required: true
+ *         description: the id for the user
+ *     responses:
+ *       200:
+ *         description: OK
+ *         content:
+ *           application/json:
+ *             schema:
+ *               oneOf:
+ *                 - type: object
+ *                   $ref: '#/definitions/FeeRecordCorrectionResponse'
```

Comment:
I imagine we will soon have a get end point for an actual correction and this name might conflict so best to keep the reference to 'transient' in it please